### PR TITLE
[RR]Update README.md

### PR DESCRIPTION
**Pull request by Ananyaiitbhilai**
created a section named how to create a PR - tells about step-by-step procedure about creating a PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,96 @@ Global Proxy Manager for android [ROOT]
...
 The success of our vision to bring knowledge to every single hand depends on you. Even a small contribution helps. All forms of contributions are highly welcomed and valued.
 
+## 📄 Creating a PR
+
+If you want to optimize this project, add features or remove bugs from this project, you are highly encouraged to do so by creating a PR. Your PR would be reviewed by the maintainer and then it depends on maintainer to merge that PR or not.
+
+If you don't know how to create a PR, don't worry. Here is a step-by-step procedure on how to proceed to create a PR.
+
+
+**#**  Take a look at the existing or create your own issues. A first good issue can be updating README file. Since Open Lake maintainers are generous enough, it is optional to create a issue or wait for a issue to be assigned to you, you can directly create a PR without creating an issue.
```
Comment (yatendra1999) [inline README.md:23]: Hmm. We are not generous in the least. LOL. Issues in my opinion are used for planned changes or mostly for bug reports from people who use the project. Also there are tags for good first issues, so you can just tell them to view by tags at first.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,96 @@ Global Proxy Manager for android [ROOT]
...
+
+```
+git checkout -b <your branch name>
+```
+
+**9.** Perform your desired changes to the code base.
+
+<p align="center"><img width=40% src="https://media.giphy.com/media/QNFhOolVeCzPQ2Mx85/giphy.gif"></p>
+
+**10.** Track your changes .
```
Comment (yatendra1999) [inline README.md:69]: Tracked and untracked changes are different. They usually refer to what github sees and what it ignores using .gitignore. The correct term would be staging the changes.

**Review by yatendra1999 — CHANGES_REQUESTED** [review]

**Comment by yatendra1999** [comment]
GIFs were good. LOL

File: README.md [hunk 3]
```diff
@@ -13,6 +13,96 @@ Global Proxy Manager for android [ROOT]
...
+
+**12.** Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your branch name>
+```
+
+**13.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
+
+**14.** Add appropriate title and description to your pull request explaining your changes. You can see the official PR Format Guidelines [here](https://github.com/OpenLake/Leaderboard-Pro/wiki/PR-Format-Guidelines).
```
Comment (yatendra1999) [inline README.md:89]: That is the format for that repo, this project doesn't have one and I am fine with people including only the necessary stuff like listing the changes made and adding a meaningful heading reflecting the changes.

**Comment by yatendra1999** [comment]
The same review is valid for your PR in the other repo as well. But this information usually resides in wiki pages. 
Also I would suggest you to try and make changes other than readme.

File: README.md (hunk 1 above)
Comment (Ananyaiitbhilai) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Ananyaiitbhilai) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Ananyaiitbhilai) [reply README.md]: done

**Comment by Ananyaiitbhilai** [comment]
> The same review is valid for your PR in the other repo as well. But this information usually resides in wiki pages.
> Also I would suggest you to try and make changes other than readme.

ok, in code contributors section of readme, i have added the link to wiki. So if feasible, u can add the instruction about creating a PR in wiki and then the wiki link would be provided in readme.
I would try but a major problem is that I have never worked with flutter previously and if a basic workflow diagram was there it would be better and there are hundreds of new words , that I have never encountered ever before. Even I didn't know about the proxy managers, its use until I saw this project

**Review by yatendra1999 — APPROVED** [review]

**Comment by yatendra1999** [comment]
> I would try but a major problem is that I have never worked with flutter previously and if a basic workflow diagram was there it would be better and there are hundreds of new words , that I have never encountered ever before. Even I didn't know about the proxy managers, its use until I saw this project
Most projects will have a steep learning curve right at the start. If you have any issues just contact people and ask them about it. This goes for all projects. You don't need to understand it all on your own and we don't expect you to.

**Pull request merged**